modules/lang-painless/src/main/java/org/elasticsearch/painless/node/SFor.java

```diff
@@ -33,18 +33,36 @@
  */
 public class SFor extends AStatement {
 
-    protected final ANode initializer;
-    protected final AExpression condition;
-    protected final AExpression afterthought;
-    protected final SBlock block;
-
-    public SFor(Location location, ANode initializer, AExpression condition, AExpression afterthought, SBlock block) {
-        super(location);
-
-        this.initializer = initializer;
-        this.condition = condition;
-        this.afterthought = afterthought;
-        this.block = block;
+    private final ANode initializerNode;
+    private final AExpression conditionNode;
+    private final AExpression afterthoughtNode;
+    private final SBlock blockNode;
+
+    public SFor(int identifier, Location location,
+            ANode initializerNode, AExpression conditionNode, AExpression afterthoughtNode, SBlock blockNode) {
```

Comment:
Is this indent correct?

Reply:
Unfortunately, it is. Without, it will exceed our characters-per-line limit.